index.js

```diff
@@ -0,0 +1,36 @@
+const formatDate = (timeInSecond) => {
+    
+    function clearHours(h) {
+        if(h == 0 || isNaN(h) == true){
```

Comment:
space after `if`, example: `if (`

Reply:
compare type and `isNaN(h)` will return true or false, so you don't need to compare it, should be:
`if (!h || isNaN(h))`